core/templates/dev/head/pages/exploration_editor/editor_tab/ResponsesService.js

```diff
@@ -178,23 +190,28 @@ oppia.factory('ResponsesService', [
           _answerGroups = AnswerGroupsCacheService.get(newInteractionId);
         } else {
           _answerGroups = [];
-          stateContentIdsToAudioTranslationsService.displayed
-            .deleteAllFeedbackContentId();
-          stateContentIdsToAudioTranslationsService.saveDisplayedValue();
         }
 
+        // Update _answerGroups audio translation according to the above
+        // changes.
```

Comment:
"above changes" can pretty much refer to any of the previous 190 lines of code and is unclear. Maybe "This is necessary in order to keep the audio translations of the answer groups in sync with the answer groups that are fetched from the cache."?

Reply:
I agree that can be misleading. Thanks for providing a clear comment!

Reply:
Done!